amgut/templates/portal.html

```diff
@@ -143,6 +156,13 @@ <h2>{% raw tl['LOG_SAMPLE_HEADER_1'] %}</h2>
             <p class="portal_blurb">{% raw tl['LOG_SAMPLE_TEXT_1'] %}</p>
             <div class="centered"><img src="{% raw media_locale['SITEBASE'] %}/static/img/labeled_tube.png"></div>
             <p class="portal_blurb">{% raw tl['LOG_SAMPLE_TEXT_2'] %}</p>
+                <h2><font color="red">{% raw tl['LOG_SAMPLE_NEW'] %}</font></h2>
+                <ul>
+                {% for barcode in barcodes %}
+                    <li><a href="{% raw media_locale['SITEBASE'] %}/authed/add_sample_overview/">{{barcode}}</a></li>
+                {% end %}
+                </ul>
+            <p class="portal_blurb">{% raw tl['LOG_SAMPLE_TEXT_3'] %}</p>
         </div>
```

Comment:
Now displays barcodes that need to be assigned. Clicking on a barcode will redirect to the add sample overview site